.github/workflows/cibuildwheel.yml

```
@@ -52,8 +65,9 @@ jobs:
           CIBW_SKIP: "cp35-* cp36-* cp39-*"
           CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
           CIBW_MANYLINUX_I686_IMAGE: manylinux1
-          CIBW_BEFORE_BUILD: pip install certifi numpy==1.16
+          CIBW_BEFORE_BUILD: pip install certifi numpy==${{ matrix.cibw_numpy_version }}
```

```suggestion
          CIBW_BEFORE_BUILD: pip install certifi oldest-supported-numpy
```

Also below. If you run into issues installing NumPy for testing, probably should make this "ALL" (but you should not, I think, as that will be the latest).